main/src/com/google/refine/util/StringUtils.java

```diff
@@ -48,4 +48,35 @@ public static String toString(Object o) {
             return o.toString();
         }
     }
+
+    /**
+     * String formatting method for the array of objects which keep the json array format
+     * @param o an array of objects to be converted to a string
+     * @return string representing array of objects
+     */
+    public static String JSONtoString(Object[] o){
```

Comment:
I think we could directly use the Jackson library for this (`ParsingUtilities.mapper.writeValueAsString` if I remember correctly).

Reply:
> I think we could directly use the Jackson library for this (`ParsingUtilities.mapper.writeValueAsString` if I remember correctly).

I think it's different. I just search for this function, it will return the JSON object to string like:
{key1: value 1,
key2: value 2}
What my function does is to return a JSON array to string like:
["value1", "value2", "value3"]

Reply:
I think the Jackson function will work both for objects (where it will indeed return things like `{"key1":"value1", "key2":"value2"}`) and arrays (where it will return things like `["value1", "value2", "value3"]`).

Reply:
> I think the Jackson function will work both for objects (where it will indeed return things like `{"key1":"value1", "key2":"value2"}`) and arrays (where it will return things like `["value1", "value2", "value3"]`).

I did more search but still did not find any information that this function can handle the array case. (I do find the ObjectMapper can read from JSON array but no information about write) I would really appreciate it if you can reference a document or tutorial to show how to do that.

Reply:
How about you just try it out?
```java
Object[] myArray = new Object[] { "foo", true, 32 };
System.out.println(ParsingUtilities.mapper.writeValueAsString(myArray));
```

Reply:
Thanks for your hint. Yes, it works. I will modify it later.